pkg/utils/log/log.go

```diff
@@ -30,8 +30,19 @@ const (
 	SpanIDField        = "span.id"
 	TraceIDField       = "trace.id"
 	TransactionIDField = "transaction.id"
+
+	testLogLevelEnvVar = "ECK_TEST_LOG_LEVEL"
 )
 
+func init() {
+	// Introduced mainly as a workaround for a controller-runtime bug.
+	// https://github.com/kubernetes-sigs/controller-runtime/issues/1359#issuecomment-767413330
+	// However, it is still useful in general to adjust the log level during test runs.
+	if logLevel, err := strconv.Atoi(os.Getenv(testLogLevelEnvVar)); err == nil {
```

Comment:
I also wonder can we not just use the regular log verbosity setting to work around this issue? `--log-verbosity`

Reply:
Unfortunately, that wouldn't work because the tests don't go through the `main` entrypoint -- where that flag is handled. 😞

Reply:
Hm, ok. But don't you have to pass on this additional environment variable through the Docker image build and/or the e2e k8s job definition for this to work in non-local e2e test runs?